src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+
+  align-items: center;
+
+  &__image {
+    box-sizing: border-box;
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    text-align: left;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+  }
+
+  &__text {
+    font-size: 12px;
+    color: $color-secondary;
+  }
+
+  &__number {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    margin-top: 16px;
+    align-items: end;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    line-height: 14px;
+  }
+
+  &__button {
+    display: flex;
+    margin-top: 16px;
+    align-items: center;
+    justify-content: center;
+
+    background-color: $color-blue;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+  }
+
+  &__link {
+    display: block;
+    border-radius: 5px;
+    margin-top: 16px;
+    padding: 12px 70px;
+    background-color: $color-blue;
+    border: 1px solid $color-blue;
+
+    text-align: center;
+    line-height: 16px;
+    font-weight: 700;
+    font-size: 14px;
+    color: #fff;
+    text-decoration: none;
+
+    &:hover {
+      border: 1px solid $color-blue;
+      background-color: #fff;
+      color: $color-blue;
+    }
+  }
+}
```

Comment:
good practice is to leave 1 line at the end of the file (please do this fix in each your file)